apps/src/gamelab/GameLab.js

```diff
@@ -1063,6 +1072,46 @@ GameLab.prototype.onP5Draw = function () {
   this.completeRedrawIfDrawComplete();
 };
 
+/**
+ * Capture a thumbnail image of the play space if the app has been running
+ * for long enough and we have not done so already.
+ */
+GameLab.prototype.captureInitialImage = function () {
+  if (!project.isOwner() || this.initialCaptureComplete || this.tickCount < CAPTURE_TICK_COUNT) {
+    return;
+  }
+  this.captureImage(THUMBNAIL_SIZE);
+  this.initialCaptureComplete = true;
```

Comment:
Waiting for an interpreter tick count would be easy in a level test, but it looks like we still don't have those for gamelab. I could test this aspect of the feature by exposing `initialCaptureComplete` via a `__TestInterface` and then have the UI test wait until it is set. I will plan to do that when I write the end-to-end UI test for this feature, which will include navigating to the teacher dashboard to see that the image appeared.